src/common/components/FriendsPagesSwitcher/index.js

```diff
@@ -9,7 +9,7 @@ import {
 } from '../../../pages/friends';
 
 function FriendsPagesSwitcher() {
-  const [value, setValue] = useContext(FriendsPagesContext);
+  const [value, _] = useContext(FriendsPagesContext);
```

Comment:
dont need to decompose the second value
`const [value] = useContext(FriendsPagesContext);`